omod/src/main/java/org/openmrs/module/registrationapp/page/controller/RegisterPatientPageController.java

```diff
@@ -62,6 +66,10 @@ public void addModelAttributes(PageModel model, Patient patient, AppDescriptor a
                 Context.getAdministrationService().getGlobalProperty("addresshierarchy.enableOverrideOfAddressPortlet", "false"));
         model.addAttribute("breadcrumbOverride", breadcrumbOverride);
         model.addAttribute("relationshipTypes", Context.getPersonService().getAllRelationshipTypes());
+
+        List<Extension> includeJavaScriptFragments = appFrameworkService.getExtensionsForCurrentUser("registerPatient.includeJavaScriptFragments");
```

Comment:
I would just call this "includeFragments", since theortically it could be used to include something other than Javascript (though it is likely 90% of the time to be Javascript... I guess custom CSS might be another use cse)